### DISCUSSION TITLE
HL7 message stuck in the queue

### DISCUSSION THREAD

**Discussion by HassanBosha** [nextgenhealthcare/connect#6164] [Q&A] [answered]
When an HL7 message fails to be sent to the destination, Mirth retries sending it. However, if it fails repeatedly, it could be due to a profile mismatch or another issue with the message. In this situation, the message becomes stuck in an infinite loop of sending attempts, which prevents other HL7 messages from being processed.

I want Mirth to skip these failed messages and continue processing the upcoming messages in the queue.

**Comment by Hueli** [comment, marked as answer]
If "Queue Message" is switched on, there is an option "Rotate Queue" under "Advance Queue Settings". This point is intended for exactly this situation. In conjunction with a value of e.g. 1 for "Retry Count Before Queue/Error" this might be the solution.
